python/spacewalk/rhn-conf/rhn_server_satellite.conf

```diff
@@ -43,3 +43,9 @@ default_mail_from =
 reposync_download_threads = 5
 reposync_timeout = 300
 reposync_minrate = 1000
+
+# URLGrabber log level. This parameter is used by spacewalk-repo-sync to provide
+# additional logs, overriding URLGRABBER_DEBUG. It takes the form "level,filename". 
```

Comment:
Sorry for bothering you with this question so late after you opened the PR, I just took another look and this comment caught my eyes. In general, I'm used to software using environment variables to override configuration.

Is the design choice to override `URLGRABBER_DEBUG` using this config deliberately taken by us, or is it "imposed onto us" by the way urlgrabber handles passing a `logspec` parameter?